backend/src/logger.ts

```diff
@@ -62,6 +62,18 @@ switch (env) {
                         destination: 1, // STDOUT
                     },
                 },
+                ...(process.env.DISCORD_WEBHOOK_URL
```

Comment:
There should be error checking here. If this environment variable is not set, this will be undefined or empty string and cause unexpected error. We should check that this variable is set and terminate the program startup otherwise